client/app/assets/less/ant.less

```diff
@@ -252,11 +252,15 @@
   z-index: 1000; // make sure it doesn't cover drawer
 }
 
-// flexible width for notifications
+// Notification overrides
 .@{notification-prefix-cls} {
   // vertical centering
   &-notice-close {
     top: 20px;
     right: 20px;
   }
+
+  &-notice-description {
+    max-width: 484px;
```

Comment:
The default is `width: 384px`, `484px` seemed a good one, but LMK if you have other suggestions.